jina/executors/indexers/keyvalue.py

```diff
@@ -75,6 +77,34 @@ def query(self, key: int) -> Optional[bytes]:
             with mmap.mmap(self.query_handler.body, offset=p, length=l) as m:
                 return m[r:]
 
+    def update(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
+        # check: hard fail (raises) on key not found
+        missed = []
+        for key in keys:
+            if self.query_handler.header.get(key) is None:
```

Comment:
Can you please add an integration test for two updates calls in a row on the same `idxer` object? I'd expect this line to fail then. If that is intended, then write a test, which expects to fail in the second `update` call in order to document, that double update on the very same `idxer` object is not possible.

Reply:
You mean like this https://github.com/jina-ai/jina/blob/e318088a30897ba3ca538de8e0f5f471af92688c/tests/unit/executors/indexers/test_binary_indexer.py#L119 ? Seems to pass

Reply:
Yes exactly. Even better.